### DISCUSSION TITLE
Can I depend on dev builds?

### DISCUSSION THREAD

**Discussion by SpectreVoid** [dmulloy2/ProtocolLib#2974] [General]
Hey can't find any information on dev builds using maven. The version does not work when I try using SNAPSHOT versions, and unsure how I can use the correct version for the latest dev build in my projects?

**Comment by Barpadzik** [comment]
Bump

**Comment by josemmo** [comment]
When I want to depend on a dev build of ProtocolLib, I use [Jitpack](https://jitpack.io/#dmulloy2/ProtocolLib) and specify the commit hash:

```xml
<repositories>
    <repository>
        <id>jitpack.io</id>
        <url>https://jitpack.io</url>
    </repository>
</repositories>

<dependency>
    <groupId>com.github.dmulloy2</groupId>
    <artifactId>ProtocolLib</artifactId>
    <version>1e3fa2d36e</version>
    <scope>provided</scope>
</dependency>
```

Some builds don't work sometimes, but for development is good enough.
